test/JDBC/input/isc-column_privileges-prepare.txt

```diff
@@ -0,0 +1,18 @@
+drop database if exists prep;
```

Comment:
we should test in master database if possible instead of custom db. Also, use test filename as prefix to avoid duplicate names for objects.